Sources/Host/Darwin/X86_64/MachX86_64.cpp

```diff
@@ -120,7 +120,7 @@ ErrorCode Mach::writeCPUState(ProcessThreadId const &ptid,
 
   mach_msg_type_number_t threadStateCount = x86_THREAD_STATE_COUNT;
   kern_return_t kret = thread_set_state(
-      thread, x86_THREAD_STATE, (thread_state_t)&state, threadStateCount);
+      thread, x86_THREAD_STATE, (thread_state_t)&threadState, threadStateCount);
```

Comment:
Aha, let me fix the readCPUState one as well then.

On Mon, May 9, 2016 at 3:00 AM, kokarez <notifications@github.com> wrote:

> In Sources/Host/Darwin/X86_64/MachX86_64.cpp
> <https://github.com/facebook/ds2/pull/327#discussion_r62475272>:
>
> > @@ -120,7 +120,7 @@ ErrorCode Mach::writeCPUState(ProcessThreadId const &ptid,
> >
> >    mach_msg_type_number_t threadStateCount = x86_THREAD_STATE_COUNT;
> >    kern_return_t kret = thread_set_state(
> > -      thread, x86_THREAD_STATE, (thread_state_t)&state, threadStateCount);
> > +      thread, x86_THREAD_STATE, (thread_state_t)&threadState, threadStateCount);
>
> Good catch, I've most likely include this issue when fix the name of the
> structure. Note that the same issue is in readCPUState :D
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/327/files/942960be6b3e6b91f6076eee6f1ab42bc11e98bd..61799dc3644b2f5376e29c2e6169ad9984777ff4#r62475272>
>



-- 
Stephane Sezer